packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -163,6 +165,7 @@ export default function NewCommentModal({
   };
   const useUserMentionOptions = useUserMention({
     postId: props.post.id,
```

Comment:
But here we still have the post as option, so why can't we abstract the `sourceId` from this?

Reply:
This is legacy code. Let me refactor it quickly.

Reply:
Oh, I thought you initially wanted to pass the whole `post` to the `useUserMention` - let me change the `sourceId` to come from here itself.